samples/counter/apps/gradle.properties

```diff
@@ -0,0 +1,2 @@
+# Basically the JS equivalent of a fat jar
+kotlin.js.ir.output.granularity=whole-program
```

Comment:
This is only needed if you are not running webpack (i.e., not running productionBrowserDistribution or whatever the task is). Webpack will always output a single file. It doesn't really matter if Kotlin spits out 1 or 100 files in that case.

Did something recommend this?

Reply:
If I don't add this, it fails to load any of the compose runtime JS files, almost as if no dependencies are included 🤔

Reply:
specifically `./gradlew :samples:counter:apps:jsBrowserRun --continuous`

Reply:
hmm, just tried again with it commented and now getting this error at build-time

```
> Task :samples:counter:apps:compileDevelopmentExecutableKotlinJs FAILED
e: java.lang.IllegalArgumentException: Cross module dependency resolution failed due to signature 'org.w3c.dom.events/EventListener|2833721539368120484[0]' redefinition
        at org.jetbrains.kotlin.ir.backend.js.transformers.irToJs.JsIrModuleCrossModuleReferenceBuilder.buildCrossModuleRefs(JsIrProgramFragment.kt:168)
        at org.jetbrains.kotlin.ir.backend.js.transformers.irToJs.CrossModuleDependenciesResolver.resolveCrossModuleDependencies(JsIrProgramFragment.kt:131)
        at org.jetbrains.kotlin.ir.backend.js.ic.JsExecutableProducer.buildMultiModuleExecutable(JsExecutableProducer.kt:56)
        at org.jetbrains.kotlin.ir.backend.js.ic.JsExecutableProducer.buildExecutable(JsExecutableProducer.kt:30)
        at org.jetbrains.kotlin.cli.js.K2JsIrCompiler.doExecute(K2JsIrCompiler.kt:309)
        at org.jetbrains.kotlin.cli.js.K2JSCompiler.doExecute(K2JSCompiler.java:181)
        at org.jetbrains.kotlin.cli.js.K2JSCompiler.doExecute(K2JSCompiler.java:72)
        at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:100)
        at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:46)
        at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:101)
        at org.jetbrains.kotlin.daemon.CompileServiceImpl.compile(CompileServiceImpl.kt:1497)

```

Reply:
darn, looks like this https://youtrack.jetbrains.com/issue/KT-60852. Needs kotlin 1.9.10